frontend/components/BaseButton.vue

```diff
@@ -0,0 +1,15 @@
+<template>
+  <button>
+    <i v-if="hasIcon" :class="icon"></i>
```

Comment:
I'm not sure, but I think it can be simplify to one props: `icon`, and then you can do: `v-if="icon ? true : false`, but as I said, I'm not sure it will work correctly with string. Please, check it.

Reply:
Okay, I'm on it, thanks!

Reply:
I tried doing It as You say here, but it doesn't seem to work correctly, or maybe I'm not doing it the right way.